tests/connect/compression-001.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\Driver\Manager: Connecting with 'noop' compressor
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+ini_set('mongodb.debug', 'stdout');
+$manager = new MongoDB\Driver\Manager(STANDALONE, [ 'compressors' => 'noop'] );
+ini_set('mongodb.debug', null);
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+%A"compressors";s:4:"noop"%A
```

Comment:
This doesn't appear to test anything more than PHPC's own `Created client hash:` debugging, so I think we can just remove this test. Even if libmongoc's URI parsing failed to parse compressors entirely, this would still pass.